sdk/translation/ai-translation-document-rest/tsconfig.json

```diff
@@ -7,11 +7,5 @@
       "@azure-rest/ai-translation-document": ["./src/index"]
     }
   },
-  "include": [
-    "./src/**/*.ts",
-    "./src/**/*.mts",
-    "./src/**/*.cts",
-    "test/**/*.ts",
-    "samples-dev/**/*.ts"
-  ]
+  "include": ["./src/**/*.ts", "./src/**/*.mts", "./src/**/*.cts", "test/**/*.ts", "./test/**/*.ts"]
```

Comment:
I looked at the few other projects and they were not building, but if it is recommended that we do, I can add it back, Please lmk

Reply:
tshy-based libraries are not building their samples because tshy builds src only so even if we include samples-dev, they wont get built. Translation libraries don't use tshy yet so this is not a concern here and it is better to type-check samples so we can catch type errors before incorrectly typed code get published.

Reply:
Added samples-dev back